requirements/deploy.txt

```diff
@@ -1,11 +1,11 @@
 -r pip.txt
-psycopg2==2.7.3.2
-gunicorn==19.1.0
-pysolr==2.0.13
-django-redis-cache==1.6.3
-django-mailgun==0.2.2
+psycopg2==2.7.5
+gunicorn==19.8.1
+pysolr==3.7.0
```

Comment:
I think we never upgraded the `deploy.txt` file. So, we should be careful with them.